lib/tasks/migrate_merits_assessment.rake

```diff
@@ -0,0 +1,9 @@
+namespace :merits_assessment_flow_steps do
+  desc 'Rename start_merits_assessments flow step to start_chances_of_successes'
+  task migrate: :environment do
+    LegalAidApplication.where(provider_step: 'start_merits_assessments').each do |app|
+      app.provider_step = 'start_chances_of_successes'
+      app.save
+    end
+  end
+end
```

Comment:
@mpw5 Is this and the rollback rake file all that's needed for now?

Assuming nothing else needed except running it manually after merging